src/Console/CreateInfrastructureFromTemplateCommand.cs

```diff
@@ -20,6 +20,8 @@ public CreateInfrastructureFromTemplateCommand(string name, string description =
             {
                 IsRequired = true
             });
+            Add(new Option<bool>(new[] { "--force", "-f", }, Resources.INFRASTRUCTURE_FORCE_HELP));
+            Add(new Option<int>(new[] { "--timeout", "-to", }, Resources.INFRASTRUCTURE_CREATION_SECONDS_TIMEOUT_HELP));
```

Comment:
the timeout shouldn't be configurable, and should be static

Reply:
@vitorpmoreira PortalSDK have a default timeout, but with this option, we can change the timeout (if necessary) to unblock the customer infrastructure. I don't know why this shouldn't be configurable.

Reply:
removed :)